analysis/src/Hover.ml

```diff
@@ -125,31 +125,44 @@ let getHoverViaCompletions ~debug ~path ~pos ~currentFile ~forHover
     ~supportsMarkdownLinks =
   match Completions.getCompletions ~debug ~path ~pos ~currentFile ~forHover with
   | None -> None
-  | Some (completions, {file; package}) -> (
+  | Some (completions, ({file; package} as full), scope) -> (
+    let rawOpens = Scope.getRawOpens scope in
+    let allFiles =
```

Comment:
Nit: there's a bit of redundancy.
`allFiles` comes from `package`.
There are functions that take both `allFiles` and `package` (or `full`). So they' don't really need all those args.
I guess `allFiles` can be either a function of `package`, or directly a field there. So it's computed on the spot when needed instead of being passed around.

Reply:
Will fix in separate PR.